compiler/luci/export/src/CircleOperationExporter.cpp

```diff
@@ -630,6 +631,20 @@ void OperationExporter::visit(luci::CircleLogSoftmax *node)
                 CreateLogSoftmaxOptions(builder).Union());
 }
 
+void OperationExporter::visit(luci::CircleMatrixSetDiag *node)
+{
+  uint32_t op_idx = md.registerBuiltinOpcode(circle::BuiltinOperator_MATRIX_SET_DIAG);
+  std::vector<int32_t> inputs_vec{get_tensor_index(node->input()),
+                                  get_tensor_index(node->diagonal())};
+  std::vector<int32_t> outputs_vec{get_tensor_index(static_cast<loco::Node *>(node))};
+  auto inputs = builder.CreateVector(inputs_vec);
+  auto outputs = builder.CreateVector(outputs_vec);
+  auto options = CreateMatrixSetDiagOptions(builder);
+  auto op_offset = CreateOperator(builder, op_idx, inputs, outputs,
+                                  circle::BuiltinOptions_MatrixSetDiagOptions, options.Union());
+  gd._operators.push_back(op_offset);
```

Comment:
Can you please use `export_simple()` ? Purpose is to decrease total LOC of this class for our SE guide; so that this class doesn't get huge.